### PR TITLE
Improve logs ui performances

### PR DESCRIPTION
**Pull request by Warry**

### CODE REVIEW

File: ui/app/assets/widgets/home/working/working.js [hunk 1]
```diff
@@ -9,40 +9,45 @@ define([
...
+
+  var State = {
+    logs: logs,
+    scroll: scroll
+  }
+
+  websocket.subscribe({type: "sbt", subType: "CoreLogEvent"}).fork().each(function(message) {
+    logs.push({
+      message: message.event.entry.message,
+      type: "info"
```
Comment (havocp) [inline ui/app/assets/widgets/home/working/working.js]: is this deliberate to always use info log level?

**Comment by havocp** [comment]
This is a nice focused PR, it looks clean.

File: ui/app/assets/widgets/home/working/working.js (hunk 1 above)
Comment (Warry) [inline ui/app/assets/widgets/home/working/working.js]: Yeah, there is no specific styling in those logs anyway.

File: ui/app/assets/plugins/run/system/system.html [hunk 2]
```diff
@@ -18,10 +18,11 @@
         </div>
       </div>
       <div class="logs" data-bind="logScroll: memoLogsScroll">
-        <ul data-bind="foreach: sbt.logs.stdout">
+        <ul data-bind="logEach: sbt.logs.stdout">
           <li data-bind="text: event.entry.message, attr: { 'data-level': event.entry.level, 'data-type': event.entry.type }"></li>
```
Comment (havocp) [inline ui/app/assets/plugins/run/system/system.html]: does event.entry.type need to be event.entry.$type

File: ui/app/assets/widgets/home/working/working.js (hunk 1 above)
Comment (havocp) [inline ui/app/assets/widgets/home/working/working.js]: event.entry.level certainly can exist in CoreLogEvent, whether you happened to get a non-info one or not...

File: ui/app/assets/plugins/run/system/system.html (hunk 2 above)
Comment (Warry) [inline ui/app/assets/plugins/run/system/system.html]: that's one of the regression I was mentioning, earlier in the call.

Comment (havocp) [inline ui/app/assets/plugins/run/system/system.html]: I think we're going to stick to $type so the whole protocol is consistent, even though it is uglier in JS. so we should go ahead and update this.

Comment (Warry) [inline ui/app/assets/plugins/run/system/system.html]: Oh sorry. gotcha !

**Pull request merged**